crates/cast/bin/tx.rs

```diff
@@ -172,6 +172,10 @@ where
             if let Some(priority_fee) = tx_opts.priority_gas_price {
                 tx.set_max_priority_fee_per_gas(priority_fee.to());
             }
+
+            if let Some(access_list) = tx_opts.access_list {
```

Comment:
there is 2930 legacy transaction with access list, so we can set it even if `legacy` is true